scranShipper/src/RestaurantDisplay/style.css

```diff
@@ -1,10 +1,17 @@
-.menuWhole {
-    width: 80%;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    align-items: flex-start;
+/*.menuWhole {*/
+/*    display: flex;*/
+/*    flex-direction: row;*/
+/*    flex-wrap: wrap;*/
+/*    justify-content: space-evenly;*/
+/*    align-items: flex-start;*/
+/*}*/
```

Comment:
have a feeling all of this should be deleted as it has been replaced with bootstrap